src/third-version.ts

```diff
@@ -391,6 +391,10 @@ function convertParameter(parameter: any): any {
     }
   }
 
+  if(parameter.schema?.$ref) {
```

Comment:
this will only log an error, but conversion of parameter continues and basically leads to broken conversion?

Reply:
Yes, didn't see any better option 🙃

Reply:
You can add an option to make it throw out errors - default will only do the logs.

Reply:
I think that:
- by default we just log, but also skip further conversion of parameter. So log must be clear that something was skipped, and what exactly and how manually fix it
- flag that allows to throw is a nice have. But if we skip, what's the point?

Reply:
> by default we just log, but also skip further conversion of parameter. So log must be clear that something was skipped, and what exactly and how manually fix it

Alright, I will change it to just be an empty parameter 🙂

Reply:
The behavior was actually just to use an empty parameter so no need for options as it is not really useful.

Added a test to show it as well.